src/test/java/me/snov/newrelic/elasticsearch/reporters/NodesStatsReporterTest.java

```diff
@@ -1,21 +1,17 @@
-package me.snov.newrelic.elasticsearch.services;
+package me.snov.newrelic.elasticsearch.reporters;
 
 import me.snov.newrelic.elasticsearch.parsers.NodesStatsParser;
-import me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter;
 import me.snov.newrelic.elasticsearch.responses.NodesStats;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.IOException;
 import java.io.InputStream;
-import java.net.URL;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertTrue;
 
 public class NodesStatsReporterTest {
 
-    private static final String nodesStatsUrl = "http://localhost:9200/_nodes/stats"; 
```

Comment:
I moved integration tests to another group: https://github.com/s12v/newrelic-elasticsearch/pull/35